app/views/govuk_component/govspeak_html_publication.raw.html.erb

```diff
@@ -2,7 +2,16 @@
   govspeak_locals = { content: content }
   govspeak_locals[:direction] = local_assigns.fetch(:direction) if local_assigns.include?(:direction)
   govspeak_locals[:rich_govspeak] = local_assigns.fetch(:rich_govspeak) if local_assigns.include?(:rich_govspeak)
+  sticky_footer_html = local_assigns.fetch(:sticky_footer_html) if local_assigns.include?(:sticky_footer_html)
 %>
-<div class="govuk-govspeak-html-publication">
+<div
+  class="govuk-govspeak-html-publication"
+  <%= "data-module=sticky-element-container" if sticky_footer_html %>
+>
   <%= render file: 'govuk_component/govspeak.raw', locals: govspeak_locals %>
+  <% if sticky_footer_html %>
```

Comment:
The module could then be limited to `government-frontend` until it's needed by more than one frontend app.